src/prog_models/prognostics_model.py

```diff
@@ -1228,24 +1245,24 @@ def generate_surrogate(self, load_functions, method = 'dmd', **kwargs):
         if 'save_pts' in config.keys():
             raise ProgModelInputException("'save_pts' is not a valid input for DMD Surrogate Model.")
 
-        if isinstance(config['inputs'], str):
-            config['inputs'] = [config['inputs']]
-        if not all([x in self.inputs for x in config['inputs']]):
-            raise ProgModelInputException(f"Invalid 'inputs' input value ({config['inputs']}), must be a subset of the model's inputs ({self.inputs}).")
+        if isinstance(config['input_keys'], str):
+            config['input_keys'] = [config['input_keys']]
+        if not all([x in self.inputs for x in config['input_keys']]):
+            raise ProgModelInputException(f"Invalid 'input_keys' value ({config['input_keys']}), must be a subset of the model's inputs ({self.inputs}).")
         
-        if isinstance(config['states'], str):
-            config['states'] = [config['states']]
-        if not all([x in self.states for x in config['states']]):
-            raise ProgModelInputException(f"Invalid 'states' input value ({config['states']}), must be a subset of the model's states ({self.states}).")
-
-        if isinstance(config['outputs'], str):
-            config['outputs'] = [config['outputs']]
-        if not all([x in self.outputs for x in config['outputs']]):
-            raise ProgModelInputException(f"Invalid 'outputs' input value ({config['outputs']}), must be a subset of the model's states ({self.outputs}).")
-
-        if isinstance(config['events'], str):
-            config['events'] = [config['events']]
-        if not all([x in self.events for x in config['events']]):
-            raise ProgModelInputException(f"Invalid 'events' input value ({config['events']}), must be a subset of the model's states ({self.events}).")
+        if isinstance(config['state_keys'], str):
+            config['state_keys'] = [config['state_keys']]
+        if not all([x in self.states for x in config['state_keys']]):
+            raise ProgModelInputException(f"Invalid 'state_keys' input value ({config['state_keys']}), must be a subset of the model's states ({self.states}).")
+
+        if isinstance(config['output_keys'], str):
+            config['output_keys'] = [config['output_keys']]
+        if not all([x in self.outputs for x in config['output_keys']]):
+            raise ProgModelInputException(f"Invalid 'output_keys' input value ({config['output_keys']}), must be a subset of the model's states ({self.outputs}).")
```

Comment:
I think "states" should be replaced with "outputs" at the end ... "must be a subset of the model's **outputs**"